[RDP] Fix "No module named 'libfb’"

laithsakka opened the pull request:
Summary:
D62215095 Introduced an import error to arvr pipelines as the is_fbcode() function does not work as intended.

This changes is_fbcode() to be a much stricter check.

Test Plan:
```
buck2 run arvr/mode/platform010/opt-stripped //arvr/libraries/depthlink/clients/mr_replay:pipeline_runner -c bolt.use_eva3_sim=True -- --config_file arvr/libraries/depthlink/clients/mr_replay/configs/runner_config.yaml --features DEPTH
```

Differential Revision: D62237502


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

malfet commented on torch/_inductor/config.py:
```diff
@@ -6,7 +6,11 @@
-    return not hasattr(torch.version, "git_version")
+    try: 
+        from libfb.py import parutil # noqa
+        return not hasattr(torch.version, "git_version")
```
Why 2nd check is needed, as `libfb.py` import will fail in all non Meta installations?
```suggestion
        return True
```

laithsakka replied:
libfb.py only available in fbcode, not avaialble in arvr.

laithsakka committed:
[RDP] Fix "No module named 'libfb’" (#135244)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/135244

D62215095 Introduced an import error to internal pipelines as the is_fbcode() function does not work as intended.

This changes is_fbcode() to be a much stricter check.

Test Plan: Run dependant internal tool

Reviewed By: aorenste

Differential Revision: D62237502
